test/src/main/java/com/distkv/drpc/test/common/Constants.java

```diff
@@ -0,0 +1,49 @@
+package com.distkv.drpc.test.common;
+
+public class Constants {
+
+  /**
+   * benchmark content.
+   */
+  public static final String TEST_CONTENT =
```

Comment:
I think we should define some contents with different sizes, like 5bytes, 100 bytes, 10m etc:

static final byte[]  TEST_CONTENT_5_BYTES = new byte[5];

static final byte[]  TEST_CONTENT_100_BYTES = new byte[100];

...

And then refine the proto to:
```proto
Message Request {
repeated byte xxx;
}
```